2020/OME_Meeting/zarr_diagram/index.html

```diff
@@ -0,0 +1,262 @@
+                left: 535px;
+                width: 112px;
+            }
+            #ome-zarr-data:hover {
+                width: 700px;
+                height: 120px;
+                left: 175px;
+            }
+            #napari {
+                top: 551px;
+                left: 551px;
+                width: 80px;
+            }
+            #napari:hover {
+                width: 750px;
+                height: 30px;
+                left: 75px;
+            }
+            #dask {
+                top: 485px;
+                left: 516px;
+                width: 148px;
+            }
+            #dask:hover {
+                top: 300px;
+                width: 780px;
+                height: 300px;
+                left: 75px;
+            }
+        </style>
+        </style>
+
+    </head>
+
+    <body style="display: flex; justify-content: center; align-items: center;">
+
+
+        <div style="width:747px; height:627px; position: relative;">
+            <img src="images/zarr-ome-diagram.png" style="position:absolute; top:0; left:0"/>
+
+            <div class="inputs">
+                <label>endpoint-url: </label>
+                <input id="endpoint" placeholder="endpoint-url" value="https://s3.embassy.ebi.ac.uk" style="width:300px" />
+                <br/>
+                <label>Image ID: </label>
+                <input id="imageid" placeholder="IMAGEID" value="6001240" style="width:150px"/>
+            </div>
+
+            <div id="bioformats2raw" class="hoverExpand">
+                <code><pre>$ build/bioformats2raw-0.2.1/bin/bioformats2raw \
+  --file_type=zarr \
+  --dimension-order=XYZCT \
+  a.fake \
+  output
+
+$ find output
+output
+output/METADATA.ome.xml
+output/data.zarr
+output/data.zarr/.zattrs
+output/data.zarr/.zgroup
+output/data.zarr/0
+output/data.zarr/0/.zattrs
+output/data.zarr/0/.zgroup
+output/data.zarr/0/0
+output/data.zarr/0/0/0.0.0.0.0
+output/data.zarr/0/0/.zarray
+output/data.zarr/0/1
+output/data.zarr/0/1/0.0.0.0.0
+output/data.zarr/0/1/.zarray
+
+$ mv output/data.zarr/0 /data/v0.1/<span class="imageId">6001240</span>.zarr
+                </pre></code>
+            </div>
+
+
+            <div id="aws" class="hoverExpand">
+                <code><pre>$ pip install awscli
+
+# upload
+aws --endpoint-url <span class='endpoint'>https://s3.embassy.ebi.ac.uk</span> cp \
+    --acl public-read --recursive \
+    /data/v0.1/<span class="imageId">6001240</span>.zarr/  \
+    s3://idr/zarr/v0.1/<span class="imageId">6001240</span>.zarr
+
+# download
+$ aws --endpoint-url --no-sign-request \
+    <span class='endpoint'>https://s3.embassy.ebi.ac.uk</span> \
+    s3 cp --recursive s3://idr/zarr/v0.1/<span class="imageId">6001240</span>.zarr \
+    /tmp/
+</pre></code>
+            </div>
+
+
+            <div id="omero-ms-zarr" class="hoverExpand">
+                <code><pre>$ git clone -b v0.1.1 git://github.com/ome/omero-ms-zarr
+$ gradle build -x test
+$ cat <<EOF > cfg
```

Comment:
I'm puzzled as to why the preview is showing me, `$ cat < cfg`.

Reply:
Probably needs to use `&lt;